ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-bgp-base.yml

```diff
@@ -1,12 +1,20 @@
 router_bgp:
   as: 65101
   router_id: 192.168.255.3
+  update: 
```

Comment:
This keyword is not consistent with your jinja template.

Reply:
Thanks and fixed